### PR TITLE
Feature/get publications

### PR DESCRIPTION
**Pull request by mdvanes**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-- 8.10
+- "8.10"
```
Comment (Hayena) [inline .travis.yml:3]: Why this change?

File: src/get-publications/index.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  await getPublications(
+    {
+      headers: {
+        origin: 'https://www.codestar.nl',
+      },
+      body: JSON.stringify({}),
+    },
+    null,
+    (_, result) => {
+      //  t.deepEqual(result, {
```
Comment (Hayena) [inline src/get-publications/index.test.js:17]: Is this comment useful to keep?

**Review by Hayena — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (mdvanes) [reply .travis.yml:3]: because it suddenly uses 8.1 if there are no quotes
